test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -212,6 +212,11 @@
     numPodsToSchedule: 2000
 - template:
     desc: MixedSchedulingBasePod
+    nodes:
+      nodeTemplatePath: config/node-default.yaml
+      labelNodePrepareStrategy:
+        labelKey: "topology.kubernetes.io/zone"
+        labelValues: ["zone1"]
```

Comment:
could you add a few more zones? If I remember correctly, the strategy supports it.

Reply:
good point, looks like the nodes are assigned a label value based on round robin.

changed all zone-based topology labelling to 3 zones